src/chrome/content/rules/Radioreference.com.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="Radioreference.com">
+	<target host="radioreference.com" />
```

Comment:
Connection times out

Reply:
I've removed the root domain, as it times out on HTTP and HTTPS.

Reply:
Thanks!